proposals/NNNN-New-vehicle-data-GearStatus.md

```diff
@@ -0,0 +1,230 @@
+# New vehicle data GearStatus
+
+* Proposal: [SDL-NNNN](NNNN-New-vehicle-data-GearStatus.md)
+* Author: [Ankur Tiwari](https://github.com/atiwari9)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core | HMI | Policy Server | SHAID | iOS | Java Suite | RPC]
+
+## Introduction
+
+This proposal is to add new vehicle data `GearStatus` to know drive mode and actual gear the transmission has selected.
+
+## Motivation
+
+In order to partner with more diverse app partners, we need to provide additional sets of vehicle data items through SDL. This goes in line with commitment to enhance SDL with even richer vehicle data content. Adding `gearDriveMode` and `actualGear` information provide more granular information on what drive mode vehicle is in and what physical gear the transmission has selected. Also we need to add more gears to the enum to support new ten speed transmissions.
+
+## Proposed Solution 
+
+We need to add `GearStatus` for `GetVehicleData`, `SubscribeVehicleData`, `UnsubscribeVehicleData` & `OnVehicleData` RPCs. Following are the changes needed in MOBILE_API and HMI_API:
+
+### Updates in MOBILE_API:
+
+#### Add to enum `VehicleDataType`: 
+
+```xml	
+<element name="VEHICLEDATA_GEARSTATUS" since="X.x"/>
+```
+#### Add new struct `GearStatus`:
+
+```xml	
+<struct name="GearStatus" since="x.x">
+	<param name="gearDriveMode" type="GearPosition" mandatory="false">
+		<description>Tells the gear drive mode i.e. PARK, DRIVE, SPORT etc</description>
+	</param>
+	<param name="actualGear" type="GearPosition" mandatory="false">
+		<description>Tells the actual gear selected by transmission</description>
+	</param>
+</struct>
+```
+#### Add new enum `GearPosition`:
+
+```xml	
+<enum name="GearPosition" since="x.x">
+	<description>The selected gear.</description>
+	<element name="PARK">
+		<description>Parking</description>
+	</element>
+	<element name="REVERSE">
+		<description>Reverse gear</description>
+	</element>
+	<element name="NEUTRAL">
+		<description>No gear</description>
+	</element>
+	<element name="DRIVE">
+	</element>
```

Comment:
Should there be a description included here?

Reply:
Yup, Added.